pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -300,6 +303,7 @@ def create_cone(self, cs_axis, position, bottom_radius, top_radius, height, name
             raise ValueError("Top radius must be greater than 0.")
         if isinstance(height, (int, float)) and height <= 0:
             raise ValueError("Height must be greater than 0.")
+        assert len(position) == 3, "``Position`` Argument must be a valid 3 Element List"
```

Comment:
Message strings should be written as complete sentences, including punctuation.

The ``position`` argument must be a valid three-element list.

The need for a leading article here is that the parameter name starts with lower case, which looks odd.